sources/packages/backend/apps/db-migrations/src/sql/CRAIncomeVerifications/Rollback-standardize-monetary-columns.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE
+    sims.cra_income_verifications
+alter column
+    reported_income type int4;
```

Comment:
Please follow the most usual types of declaration.
For `int4` it would be 'INT'.

Reply:
updated